WordPress/Sniffs/Theme/RestrictedPHPFunctionsSniff.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * WordPress Coding Standard.
+ *
+ * @package WPCS\WordPressCodingStandards
+ * @link    https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+ * @license https://opensource.org/licenses/MIT MIT
+ */
+
+/**
+ * Forbids usage of certain fuctions and recommends alternatives.
+ *
+ * @package WPCS\WordPressCodingStandards
+ *
+ * @since   0.xx.0
+ */
+class WordPress_Sniffs_Theme_RestrictedPHPFunctionsSniff extends WordPress_Sniffs_Functions_FunctionRestrictionsSniff {
```

Comment:
This WPCS has since the original PR been deprecated. The `WordPress\AbstractFunctionRestrictionsSniff` should be used instead.